frontend/src/app/main/ui/components/radio_buttons.cljs

```diff
@@ -61,12 +63,13 @@
 (mf/defc radio-buttons
   {::mf/wrap-props false}
   [props]
-  (let [children  (unchecked-get props "children")
-        on-change (unchecked-get props "on-change")
-        selected  (unchecked-get props "selected")
-        name      (unchecked-get props "name")
-        class     (unchecked-get props "class")
-        wide      (unchecked-get props "wide")
+  (let [children     (unchecked-get props "children")
+        on-change    (unchecked-get props "on-change")
+        selected     (unchecked-get props "selected")
+        name         (unchecked-get props "name")
+        class        (unchecked-get props "class")
+        wide         (unchecked-get props "wide")
+        allow-empty? (unchecked-get props "allow-empty?")
```

Comment:
Please, don't use the `?` char on props on components that can be candidate to be used externally  (example: storybook)